user_templates/mcu_defaults/main/BTT_SKR_E3mini_v2.cfg

```diff
@@ -52,4 +52,3 @@ uart_address: 1
 uart_pin: TMCUART
 tx_pin: TMCTX
 uart_address: 3
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates from the install script